tools/sync_test_server/start_server.sh

```diff
@@ -34,13 +34,16 @@ adb reverse tcp:8888 tcp:8888 || { echo "Failed to reverse adb port." ; exit 1 ;
 # Make sure that Docker works correctly with Github Docker Registry by logging in
 docker login docker.pkg.github.com -u $GITHUB_DOCKER_USER -p $GITHUB_DOCKER_TOKEN
 
+SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+# Create app configurations
+APP_CONFIG_DIR=`mktemp -d`
```

Comment:
I guess this is a bit annoying if you would like to inspect the running setup, but I see the point of not running on top of previous garbage. Maybe just do a local temp dir so it is easier to find.